docs/reference/docs/reindex.asciidoc

```diff
@@ -1119,6 +1122,32 @@ POST _reindex
 // TEST[s/otherhost:9200/\${host}/]
 // TEST[s/\.\.\.,/"username": "test_admin", "password": "x-pack-test-password",/]
 
+[[CCS-based-reindex-from-remote]]
+==== CCS based reindex from remote
+
+Reindex supports CCS based reindexing from a remote Elasticsearch cluster:
+
+[source,console]
+--------------------------------------------------
+POST _reindex
+{
+  "source": {
+    "index": "cluster_one:my-index-000001"
+  },
+  "dest": {
+    "index": "my-new-index-000001"
+  }
+}
+--------------------------------------------------
+// TEST[setup:my_index_big]
+
+The `index` parameter must be specified with the remote cluster name and the index name, separated by 
+a colon (e.g., cluster_one:my-index-000001).
+Additionally, you must configure the remote cluster in advance and retrieve the remote cluster name.
+For more details, refer to the reference for <<add-remote-clusters>>.
+
+NOTE: Needs v8.12.0 or later on the reindexing cluster.
```

Comment:
This is insinuated by the documentation update in only recent versions, and the changelog in 8.12. So please remove this line.

Reply:
[omit note](https://github.com/elastic/elasticsearch/pull/117154/commits/e0586516ce1d8415759d7ffbb1a97c6021b880be)